packages/nimble-blazor/NimbleBlazor/DateTextColumnTypes.cs

```diff
@@ -0,0 +1,276 @@
+﻿// Suppressing rule "Identifiers should not contain type names" because "long" and "short" come from the underlying API
+#pragma warning disable CA1720
+namespace NimbleBlazor;
+
+public enum DateTextFormat
+{
+    Custom
```

Comment:
Shouldn't this enum also include `Default`?

Reply:
The default is to not specify a value, so I didn't think I needed it as an enum value. But I guess I can just add a `Default` that maps to `null`.

Reply:
`Default` mapping to `null` is what I'd expect. I believe that's what we do with our other enums in Blazor.